tools/frag4feature.xml

```diff
@@ -1,4 +1,5 @@
-<tool id="frag4feature" name="frag4feature" version="0.0.20">
+<tool id="frag4feature" name="frag4feature" version="0.0.19">
```

Comment:
@jsaintvanne Fix this: keep the original version numbers. The main author will decide at the time because at each rebase you will have conflit here.